common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -1865,7 +2139,17 @@ class NativeAuthMsalController : BaseNativeAuthController() {
                     tokenApiResult = this
                 )
             }
-            is SignInTokenApiResult.CodeIncorrect, is SignInTokenApiResult.MFARequired,
+            is SignInTokenApiResult.MFARequired -> {
+                SignInCommandResult.MFARequired(
```

Comment:
I don't know if this the right place, but when the "getAccessToken()" function calls the /token endpoint and it returns an MFA required error, what message do we provide? I believe it would be helpful to include a custom message advising the external developer to sign out and sign in again.

Reply:
This is an open point, which is captured by https://identitydivision.visualstudio.com/Engineering/_workitems/edit/3015026 and requires PM input.